File: src/LPII/Atividade2/Exercicio1.java
package LPII.Atividade2;

import java.util.Scanner;

public  class  Exercicio1 {
    public static void main(String[] args) {
        Scanner ler = new Scanner(System.in);

        int i;
        int pares = 0;
        int[] vetor = new int[20];

        System.out.println("Digite 20 numeros inteiros");

        for (i=0; i<20; i++) {
            System.out.println("Digite um valor");
            vetor[i] = ler.nextInt();
        }

        System.out.println("Numeros = ");

        for (i=0; i<20; i++) {
            System.out.println(" " + vetor[i]);
        }

        for (i=0; i<20; i++) {
            if (vetor[i] % 2 == 0) {
                pares++;
            }
        }

        System.out.println("Quantidade de pares = " + pares);

    }
}

File: src/LPII/Atividade1/Exercicio1.c
package LPII.Atividade1;

public class Exercicio1 {
    #include <stdio.h>
    #define v 20

    int main(){

        int cont, i=0, vetor[v];

        printf("Digite 20 numeros inteiros\n \n");

        for(cont=0; cont<20; cont++){
            printf("Digite o %d numero\n", cont+1);
            scanf("%d", &vetor[cont]);
        }

        printf("\n\tNUMEROS:\n");

        for(cont=0; cont<20; cont++){
            printf("\n\t%d", vetor[cont]);
        }

        for(cont=0; cont<20; cont++){
            if(vetor[cont]%2==0){
                i++;
            }
        }
        printf("\n \n Pares = %d\n", i);

        return 0;
    }
}
